.github/workflows/directoriesFilesChangePR.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         uses: AdoptOpenJDK/run-aqa@v1
         with:
           build_list: ${{ matrix.build_list }}
-          target: '_sanity.${{ matrix.build_list }}'
+          target: '_sanity.${{ matrix.build_list }}.regular'
```

Comment:
I believe target `_sanity.regular` should be enough. Parameter `build_list` has already limited the tests category.

Reply:
Yes, though there is nothing wrong with being very explicit here (for clarity to others who do not know that if build_list is set, that TKG will only build and execute the targets within that directory).

Reply:
updated using `target: '_sanity.regular'`.